tests/zfs-tests/tests/functional/cli_root/zfs_destroy/zfs_destroy_008_pos.ksh

```diff
@@ -38,14 +38,27 @@
 #
 ################################################################################
 
+function test_s_run
+{
+	typeset snap=$1
+
+    log_must $ZFS destroy -d $snap
+    log_mustnot datasetexists $snap	
+}
+
 log_assert "'zfs destroy -d <snap>' destroys snapshot if there is no clone"
 log_onexit cleanup_testenv
 
 setup_testenv snap
 
 for snap in $FSSNAP $VOLSNAP; do
-    log_must $ZFS destroy -d $snap
-    log_mustnot datasetexists $snap
+    if [[ $snap == $VOLSNAP ]]; then
+		if is_global_zone; then
+			test_s_run $snap
+		fi
+	else
+		test_s_run $snap
+	fi
```

Comment:
At the moment under Linux is_global_zone should always be true. I agree it. but from the purpose of zfs_destroy_common.kshlib designers, is_global_zone may be false, that is need to differentiate between true or false, so in order to code logic is normal, to distinguish in the footsteps of independent.
Therefore, When we zfstestsuite test VOL or VOL snapshot or VOL clone, must consider is_global_zone value,